openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -156,6 +156,12 @@ jQuery(function () {
         $('#cboxSlideshow').attr({'aria-label': 'Slideshow button', 'aria-hidden': 'true'});
     }
 
+    //conditionally load javascript for the wikipedia citation button
+    if (document.getElementById('wikilink')) {
+        import('./wikipedia_citation_button')
```

Comment:
If an element doesn't match and ID the focus method doesn't get bound to anything. This module is so small, we might as well run the line unconditionally.

Reply:
Fixed!